charts/etcd/templates/etcd-configmap.yaml

```diff
@@ -53,6 +53,9 @@ data:
     # Initial cluster state ('new' or 'existing').
     initial-cluster-state: {{ .Values.etcd.initialClusterState }}
 
+    # Initial cluster
```

Comment:
I think you can delete this and the entire file as you have switched to a `component`, right?